plugins/panels/similarity_search/operators.py

```
@@ -0,0 +1,260 @@
+                query = self._compute_combined_query(
+                    dataset,
+                    brain_key,
+                    query,
+                    negative_query_ids,
+                    patches_field=patches_field,
+                )
+
+            ctx.set_progress(0.3, label="Running similarity query...")
+
+            # Build kwargs, omitting None values
+            kwargs = dict(brain_key=brain_key)
```

Ruff (0.15.5)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)